### ISSUE TITLE
Crashes on Linux

### ISSUE THREAD

**Issue by Ikaxas1** [fonol/anki-search-inside-add-card#203]
I am using the unofficial Snap package (anki-woodrow, 2.1.35) on Ubuntu 20.04. (This is because the .deb is no longer updated and this addon is not compatible with 2.1.15.) 

Addon works fine until I try to actually read a note (e.g. clicking "Read first in queue" or going to the queue manager and trying to open a note from there). When I try, Anki immediately crashes. I have 4 notes so far, it happens on all of them. 3 are PDFs (including one book-length PDF) and one is a markdown-imported webpage for which I also imported the webpage as a PDF. 

May attempt more testing later, but right now I'm tired and going to bed.

**Comment by fonol** [comment]
Hm hard to guess what's going wrong. Anything suspicious in the terminal if you run Anki from it?
Could it be be a permission issue?

**Comment by Ikaxas1** [comment]
I ended up switching to the official Anki package, distributed via tarball from their website rather than from the repos. It works fine now. My guess now is that the problem is not with your addon. A couple possibilities: 

- Missing libraries in anki-woodrow. From the official Anki page: "Some Linux distributions include Anki in their repositories. We've seen many users experience problems with these distributed versions, due to them being out of date or missing appropriate libraries. For this reason, the compiled package above is recommended over using the version of Anki bundled with your Linux distribution." 
- I was using 2.1.35 with anki-woodrow, but when I switched to the tarball I went ahead and used 2.1.38. Perhaps your addon only works with 2.1.38. 
- When I installed anki-woodrow, I hadn't uninstalled the 2.1.15 .deb. Before I installed the tarball. I removed both other versions of Anki. Perhaps there was some conflict when I had both installed (though this seems unlikely, as I think snaps are pretty self-contained, right?)

Anyway, I'm not going to pursue this any further or do any experimentation to decide between these hypotheses if that's alright. So I'll close the issue.
